packages/iris-grid/src/FilterInputField.scss

```diff
@@ -56,13 +56,29 @@
     }
 
     &.iris-grid-has-filter {
-      // same as filter-bar-expanded-active-bg
-      background: scale-color($primary, $saturation: -75%, $lightness: -60%);
+      // This is a hack to get the filter bar inputs to match the canvas rendered equivalent
+      // --dh-color-filter-bar-expanded-active-bg relies on transparency of being drawn overtop of
+      // the grid background. Since we can't do that here, we need to mimic the effect by stacking
+      // backgrounds using a single color linear gradients as a hack. Performance is not an issue
+      // since it only applies to the selected filter bar cell.
+      background: linear-gradient(
```

Comment:
I added a similar `solid-gradient` helper for this stacking hack as part of the inline SVGs work. Not sure if it can be re-purposed for this as well.

Reply:
I feel like the less scss we use, the easier it will be to switch off it someday.